pyproject.toml

```diff
@@ -42,6 +42,8 @@ pyyaml = "*"
 starlette = "*"
 typing-extensions = "*"
 requests = { version = "*", optional = true }
+typer = "^0.4.2"
+cookiecutter = "^2.1.1"
```

Comment:
I believe we were going to make these optional? Also do they need to be pinned so aggressively, or at all?